presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1177,10 +1178,12 @@ private void analyzeWindowFunctions(QuerySpecification node, List<Expression> ou
                 }
 
                 for (Expression expression : window.getPartitionBy()) {
+                    analyzeExpression(expression, sourceScope);
```

Comment:
The reason I added this is so that `GroupingOperation` nodes having more than 63 columns can be found and a `SemanticException` can be thrown. I am basically piggybacking off of the infrastructure provided by `ExpressionAnalyzer`.

But it turns out you're right and I didn't realize that the `partition` and `sortItems` of the window function already have been analyzed. I've removed the two lines.